src/plugins/KeyboardEvents.js

```diff
@@ -223,18 +223,18 @@ module.exports = class KeyboardEvents {
 
         if (!this.isListenerAdded) {
             // This avoid having continuously pressed keys because of alt+tab or any other command that blur from tab
-            window.addEventListener('blur', this.cancelAllPressedKeys.bind(this));
+            this.removeBlurListener = this.instance.addListener(window, 'blur', this.cancelAllPressedKeys.bind(this));
 
             if (!this.keyboardCallbacks) {
-                this.keyboardCallbacks = new Map([
-                    ['keypress', this.onKeyPress.bind(this)],
-                    ['keydown', this.onKeyDown.bind(this)],
-                    ['keyup', this.onKeyUp.bind(this)]
-                ]);
+                this.keyboardCallbacks = [
```

Comment:
maybe declare this arr on constructor, init with an empty array and check if (arr.length) {}